core/common/src/PlanarClipMask.ts

```diff
@@ -150,8 +150,8 @@ export interface PriorityPlanarClipMaskArgs extends BasicPlanarClipMaskArgs {
 export class PlanarClipMaskSettings {
   /** Specifies how the mask geometry is produced. */
   public readonly mode: PlanarClipMaskMode;
-  /** For any mode other than [[PlanarClipMaskMode.Priority]], the Ids of the [GeometricModel]($backend)s containing the geometry used to produce the mask.
-   * If `undefined`, the set of all models in the view's [ModelSelector]($backend) is used.
+  /** For any mode other than [[PlanarClipMaskMode.Priority]], the Ids of the [GeometricModel]($backend)s containing the geometry used to produce the mask,
+   * and if `undefined`, no mask geometry will be created.
```

Comment:
Did you change it to produce no mask if modelIds is undefined, or is that the way it already worked in V1?
If the latter, how did you decide that we want the current behavior vs the advertised behavior?

Reply:
We modified the comment, and tried to clarify it, to match what the actual behavior already was.  The comment was unclear on what cases the undefined applied to - It is undefined in Priority Mode by definition (and does use all the models in the view in that case), but for the rest if undefined it produces no mask.  Since the code has been this way apparently forever, we thought it best to clarify/change the comment as someone may already depend on actual behavior.